sc/social/like/utils.py

```diff
@@ -128,3 +128,11 @@ def get_valid_objects(brains):
             logger.warn(msg.format(b.getPath()))
             continue
         yield obj
+
+
+def get_path_to_virtual_path(request, item):
```

Comment:
you should test just the final result as in the test I uncommented in the behavior.